src/Sylius/Bundle/UiBundle/Resources/views/Form/theme.html.twig

```diff
@@ -1,43 +1,4 @@
-{% extends 'form_div_layout.html.twig' %}
```

Comment:
Can you onboard me with these changes? Linter seems to be complaining about the lack of existence of some functions, but I need help finding them here. You've also moved some parts of the forms to another file. Isn't it a BC-Break?

Reply:
I don't think so, the current layout extend a new one which can be use on a Symfony/Monofony project without overriding it.

No CI is green now, the goal for now is to check no dependency is needed for this new layout only.